_includes/templates/summaries/linkedin.html

```diff
@@ -0,0 +1,16 @@
+<h2 class="h4">
+    <span class="cf-icon cf-icon-linkedin-square"></span>
+    Follow us on LinkedIn
```

Comment:
Updated to `Follow Us on Linkedin` in https://github.com/cfpb/cfgov-refresh/commit/ed4e0c971c8550360bf26cdfd3d295f7e5ce4ff7